volttrontesting/utils/platformwrapper.py

```diff
@@ -547,9 +547,16 @@ def startup_platform(self, vip_address, auth_dict=None, use_twistd=False,
         # A None value means that the process is still running.
         # A negative means that the process exited with an error.
         assert self.p_process.poll() is None
-        if self.message_bus == 'rmq':
-            # give some extra time for volttron process to startup
-            gevent.sleep(4)
+
+        # Check for VOLTTRON_PID
+        sleep_time = 0
+        while (not utils.is_volttron_running(self.volttron_home)) and sleep_time < 60:
```

Comment:
Why not modify is_volttron_running() of the platform wrapper and use that instead?